pkg/ddevapp/config.go

```diff
@@ -3,19 +3,20 @@ package ddevapp
 import (
 	"bytes"
 	"fmt"
-	"github.com/Masterminds/sprig/v3"
-	"github.com/drud/ddev/pkg/dockerutil"
-	"github.com/drud/ddev/pkg/globalconfig"
-	"github.com/drud/ddev/pkg/nodeps"
-	"github.com/drud/ddev/pkg/versionconstants"
-	copy2 "github.com/otiai10/copy"
 	"os"
 	"path/filepath"
 	"sort"
 	"strings"
 	"text/template"
 	"time"
 
+	"github.com/Masterminds/sprig/v3"
+	"github.com/drud/ddev/pkg/dockerutil"
+	"github.com/drud/ddev/pkg/globalconfig"
+	"github.com/drud/ddev/pkg/nodeps"
+	"github.com/drud/ddev/pkg/versionconstants"
+	copy2 "github.com/otiai10/copy"
+
```

Comment:
Changing vcode's handling go code is *hard*.  It's using official golang language tools under the hood, which is not readily configurable. If the imports in config.go are getting reorganized, you're basically fighting city hall. 

I can do manual merges with git to prevent this, if you're really committed to the current ordering.

Reply:
You don't have to change it but no, gofmt doesn't automatically do this.